lib/Dashboard/Controller/API/Jobs.pm

```diff
@@ -81,6 +81,33 @@ sub modify ($self) {
   $self->render(json => {message => 'Ok'});
 }
 
+sub show_remarks ($self) {
+  my $openqa_job_id   = $self->param('job_id');
+  my $internal_job_id = $self->jobs->internal_job_id($openqa_job_id);
+  return $self->render(json => {error => "openQA job ($openqa_job_id) does not exist"}, status => 404)
+    unless $internal_job_id;
+  my $incidents = $self->app->incidents;
+  my $incident  = sub ($remark) {
+    return undef unless my $incident_id = delete $remark->{incident_id};
+    return $incidents->number_for_id($incident_id);
+  };
+  my $remarks = $self->jobs->remarks($internal_job_id)->map(sub { {text => $_->{text}, incident => $incident->($_)} });
+  $self->render(json => {remarks => $remarks->to_array});
+}
+
+sub update_remark ($self) {
+  my $incident_number = $self->param('incident_number');
+  my $incident_id     = defined $incident_number ? $self->app->incidents->id_for_number($incident_number) : undef;
+  my $openqa_job_id   = $self->param('job_id');
+  my $internal_job_id = $self->jobs->internal_job_id($openqa_job_id);
+  return $self->render(json => {error => "openQA job ($openqa_job_id) does not exist"}, status => 404)
```

Comment:
It's ok to use blank lines to make the code a little easier to read for old eyes like mine. 😉